MARIE_AND_BERT/readme.md

```diff
@@ -74,6 +80,22 @@ and uncomment the line
 ```python
     # app.run(host='0.0.0.0', debug=True, port=5003)
 ```
+### Recreation of the required files 
+To recreate the files in `CrossGraph.zip`, please read [Dataset creation](KGToolbox/readme.md) section `CrossGraph` to create the required files for training models and  
+follow [Model training](Training/readme.md) to train the models. 
+
+To recreate the files in `Dictionaries.zip`, please read [Dictionary creation](KGToolbox/readme.md) section `Dictionaries (DATA/Dictionaries folder)` to 
+create the dictionary files. 
+
+For deployment purpose, please 
+zip the folder to create the `CrossGraph.zip` file. 
+ 
+To recreate the files in `EntityLinking.zip`, please read . For deployment purpose, zip 
```

Comment:
Please read what? Please add.

Reply:
We have pushed the updates, the information is added